sdk/keyvault/azure-security-keyvault-administration/README.md

```diff
@@ -53,71 +50,35 @@ add the direct dependency to your project as follows.
 
-The `DefaultAzureCredential` way of authentication by providing client secret credentials is being used in this getting started section but you can find more ways to authenticate with [azure-identity][azure_identity].
+The `DefaultAzureCredential` way of authentication with a system-assigned managed identity is being used in this getting started section but you can find more ways to authenticate with [azure-identity][azure_identity].
 
-#### Create/Get credentials
-To create/get client secret credentials you can use the [Azure Portal][azure_create_application_in_portal], [Azure CLI][azure_keyvault_cli_full] or [Azure Cloud Shell][azure_cloud_shell].
+#### Give a managed identity access to your managed HSM
+To give a managed identity access to your managed HSM you can use the [Azure Portal][azure_portal_managed_identity], [Azure CLI][azure_cli_managed_identity] or [Azure Cloud Shell][azure_cloud_shell].
 
-Here is an [Azure Cloud Shell][azure_cloud_shell] snippet below to
-
- * Create a service principal and configure its access to Azure resources:
- 
-```bash
-az ad sp create-for-rbac -n <your-application-name> --skip-assignment
-```
+The following example shows how to do this using the [Azure Cloud Shell][azure_cloud_shell]: 
 
-Output:
-
-```json
-{
-    "appId": "generated-app-ID",
-    "displayName": "some-app-name",
-    "name": "https://some-app-name",
-    "password": "random-password",
-    "tenant": "tenant-ID"
-}
-```
-
-* Take note of the service principal objectId
-```bash
-az ad sp show --id <appId> --query objectId
-```
-
-Output:
-```
-"<your-service-principal-object-id>"
-```
-
-* Use the returned credentials above to set the **AZURE_CLIENT_ID** (appId), **AZURE_CLIENT_SECRET** (password), and **AZURE_TENANT_ID** (tenantId) environment variables. The following example shows a way to do this in Bash:
-
-```bash
-export AZURE_CLIENT_ID="generated-app-ID"
-export AZURE_CLIENT_SECRET="random-password"
-export AZURE_TENANT_ID="tenant-ID"
-```
-
-* Create the Managed HSM and grant the above mentioned service principal authorization to perform administrative operations on the Managed HSM (replace `<your-resource-group-name>` and `<your-key-vault-mhsm-name>` with your own, unique names and `<your-service-principal-object-id>` with the value from above):
+* Create the Managed HSM and grant the above mentioned managed identity authorization to perform administrative operations on the Managed HSM (replace `<your-resource-group-name>` and `<your-managed-hsm-name>` with your own, unique names):
     
 ```bash
-az keyvault create --hsm-name <your-key-vault-mhsm-name> --resource-group <your-resource-group-name> --administrators <your-service-principal-object-id> --location <your-azure-location>
+az keyvault create --hsm-name "<your-managed-hsm-name>" --resource-group "<your-resource-group-name>" --administrators "<your-managed-identity-principal-id>" --location "<your-azure-location>"
```

Comment:
My guess is the SDK readme has these instructions because there wasn't a place to link to for customers until now. The doc you linked to is relatively new. However I agree it might be a good time to move away from sharing these instructions.

I do not see other SDKs talking about resource creation as part of data-plane SDKs especially if the data-plane SDK does not support the creation APIs.

Reply:
@heaths is there a historic reason why we add these instructions to the readme?

Reply:
I would prefer linking to the docs as well. I'm not sure what the original reason was to tell people how to create resources, it's been like this since I joined the v-team in early 2020.

Reply:
Sounds like a plan :)